ee/tables/sleeper/table.go

```diff
@@ -0,0 +1,57 @@
+package sleeper
+
+import (
+	"context"
+	"log/slog"
+	"strconv"
+	"time"
+
+	"github.com/kolide/launcher/ee/agent/types"
+	"github.com/kolide/launcher/ee/tables/tablehelpers"
+	"github.com/kolide/launcher/ee/tables/tablewrapper"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+type Table struct {
+	slogger *slog.Logger
+}
+
+func TablePlugin(flags types.Flags, slogger *slog.Logger) *table.Plugin {
+
+	columns := []table.ColumnDefinition{
+		table.IntegerColumn("duration"),
+	}
+
+	t := &Table{
+		slogger: slogger.With("table", "kolide_deadly_sleeper"),
+	}
+
+	return tablewrapper.New(flags, slogger, "kolide_deadly_sleeper", columns, t.generate, tablewrapper.WithTableGenerateTimeout(10*time.Minute))
```

Comment:
I wrapped it, to ensure traces. Even though this should only show up in debugging, I wanted to make sure we saw it if someone calls it in prod.